menu-screen.lua

```diff
@@ -136,6 +146,44 @@ function MenuScreen:drawPlayerScreens()
 	end
 end
 
+function generateName(dict)
+	local n1 = math.random(1, dict:dictLength())
+	local n2 = math.random(1, dict:dictLength())
+	return firstToUpper(dict:get()[n1]) .. ' ' .. firstToUpper(dict:get()[n2])
+end
+
+function updateJSTimer(dt)
+	if joysticks ~= nil then
+		table.foreach(joysticks, function (i)
+			joysticks[i].timer = joysticks[i].timer + dt
+		end)
+	end
+end
+
+function resetJSTimer()
+	if joysticks ~= nil then
+		table.foreach(joysticks, function (i)
+			joysticks[i].timer = 0
+		end)
+	end
+end
+
+function allJSTimerAbove(num)
+	local bool = true
+	if joysticks ~= nil then
+		table.foreach(joysticks, function (i)
+			if joysticks[i].timer < num then
+				bool =  false
+			end
+		end)
+	end
+	return bool
+end
+
+function firstToUpper(str)
```

Comment:
it has to do strictly with dictionary functionality and is only ever called in that context